lib/traject/pdc_describe_indexing_config.rb

```diff
@@ -342,38 +342,22 @@
 # to_field 'mediator_ssim', extract_xpath("/item/metadata/key[text()='dcterms.mediator']/../value")
 # to_field 'source_ssim', extract_xpath("/item/metadata/key[text()='dcterms.source']/../value")
 
-# # ==================
-# # Store all files metadata as a single JSON string so that we can display detailed information for each of them.
-# to_field 'files_ss' do |record, accumulator, _context|
-#   dataspace_handle = record.xpath('/item/handle/text()').text
-#   bitstreams = record.xpath("/item/bitstreams").map do |node|
-#     {
-#       name: node.xpath("name").text,
-#       description: node.xpath("description").text,
-#       format: node.xpath("format").text,
-#       size: node.xpath("sizeBytes").text,
-#       mime_type: node.xpath("mimeType").text,
-#       sequence: node.xpath("sequenceId").text,
-#       bundle_name: node.xpath("bundleName").text,
-#       handle: dataspace_handle
-#     }
-#   end
-
-#   # Only files in the ORIGINAL bundle in DataSpace need to be indexed,
-#   # the rest are files that have been deleted or have purposes outside
-#   # of PDC Discovery (e.g. extracted text).
-#   #
-#   # We also explicitly exclude DSpace license.txt file since it's a DSpace
-#   # generated file and we don't want it in PDC Discovery.
-#   bitstreams.reject! do |bitstream|
-#     bitstream[:bundle_name] != "ORIGINAL" || bitstream[:name] == "license.txt"
-#   end
-
-#   accumulator.concat [bitstreams.to_json.to_s]
-# end
+# ==================
+# Store all files metadata as a single JSON string so that we can display detailed information for each of them.
+to_field 'files_ss' do |record, accumulator, _context|
+  # TODO: Remove prefix once we export the complete URI in PDC Describe
```

Comment:
This issue will take care of the TODO https://github.com/pulibrary/pdc_describe/issues/902

Reply:
Is there a ticket for updating the todo, or should we wait on this PR?

Reply:
I am planning on making the change as I work on 902.